docs/sources/next/javascript-api/k6-browser/elementhandle/setchecked.md

```diff
@@ -0,0 +1,69 @@
+---
+title: 'setChecked(checked[, options])'
+description: 'Browser module: elementHandle.setChecked method'
+---
+
+# setChecked(checked[, options])
+
+{{< admonition type="warning" >}}
+
+Use [`locator.setChecked(checked [, options])`](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-browser/locator/setchecked/) instead.
+
+{{< /admonition >}}
+
+Use this method to select or unselect an `input` or `radio` checkbox.
+
+<TableWithNestedRows>
+
+| Parameter           | Type    | Default | Description                                                                                                                                                                                                                                                                                                         |
+| ------------------- | ------- | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| options             | object  | `null`  | Optional parameters.                                                                                                                                                                                                                                                                                                |
+| options.force       | boolean | `false` | Setting this to `true` will bypass the actionability checks (`visible`, `stable`, `enabled`).                                                                                                                                                                                                                       |
+| options.noWaitAfter | boolean | `false` | If set to `true` and a navigation occurs from performing this action, it will not wait for it to complete.                                                                                                                                                                                                          |
```

Comment:
```suggestion
| options.noWaitAfter | boolean | `false` | If set to `true` and a navigation occurs from performing this action, the script won't wait for it to complete.                                                                                                                                                                                                          |
```

I think this would be easier to read if we replace "it will" by either "k6" or "the script", whichever makes the most sense.